distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -56,6 +56,22 @@ type Connection
     close =
         self.connection_resource . finalize
 
+    databases : [Text]
+    databases =
+        here.wrap_sql_errors <|
+            self.connection_resource.with connection->
+                metadata = connection.getMetaData
+                schema_result_set = metadata.getCatalogs
+                Vector.Vector (JDBCProxy.getStringColumn schema_result_set "TABLE_CAT")
+
+    schemas : [Text]
+    schemas =
+        here.wrap_sql_errors <|
+            self.connection_resource.with connection->
+                metadata = connection.getMetaData
+                schema_result_set = metadata.getSchemas
+                Vector.Vector (JDBCProxy.getStringColumn schema_result_set "TABLE_SCHEM")
```

Comment:
`TABLE_SCHEM` is not a typo, right?

Reply:
yes the standard column name for getSchema.